### ISSUE TITLE
User story - Benefits on Job Post

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#36]
### If applicable, add a related Epic.

Epic: #28 

### User Story

As a **user**, I can **see some job post benefits listed** so that I can **have greater decision making power to see if I want to go for the job**.

### Acceptance Criteria

- The benefits should be unique (like tech)
- Benefits should be selectable from a dropdown
- If a benefit is not there, it appears greyed out until approved

### Please ensure you've completed these tasks before marking the issue as done.

- [ ] Reviewed the user story for clarity and completeness.
- [ ] Identified and linked related issues or epics.
- [ ] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [ ] Updated the issue with any relevant notes or feedback (to date).
- [ ] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [ ] Create a benefits table
- [ ] Establish one to many relationship between post and benefits
- [ ] Update job post creation and edit forms to include benefits
